blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -1,105 +1,53 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { useFusionContext } from 'fusion:context';
-import getProperties from 'fusion:properties';
-import getThemeStyle from 'fusion:themes';
-import { useContent, useEditableContent } from 'fusion:content';
-import styled from 'styled-components';
-import { Image, LazyLoad, isServerSide } from '@wpmedia/engine-theme-sdk';
-import { imageRatioCustomField, ratiosFor } from '@wpmedia/resizer-image-block';
-import getPromoContainer from './_children/promo_container';
-import getPromoStyle from './_children/promo_style';
+import { useEditableContent } from 'fusion:content';
+import { LazyLoad, isServerSide } from '@wpmedia/engine-theme-sdk';
+import { imageRatioCustomField } from '@wpmedia/resizer-image-block';
+import {
+  PromoHeadline, PromoImage, SmallPromoContainer, SmallPromoStyles,
+} from '@wpmedia/shared-styles';
 
-import '@wpmedia/shared-styles/scss/_small-promo.scss';
-
-const HeadlineText = styled.h2`
-  font-family: ${(props) => props.primaryFont};
-`;
-
-const SmallManualPromoItem = ({ customFields = {} }) => {
-  const { showImage = true } = customFields;
-  const { arcSite, isAdmin } = useFusionContext();
+const SmallManualPromoItem = ({ customFields }) => {
   const { searchableField } = useEditableContent();
-  const imagePosition = customFields?.imagePosition || 'right';
-
-  const headlineMarginClass = getPromoStyle(imagePosition, 'headlineMargin');
-
-  const imageMarginClass = getPromoStyle(imagePosition, 'margin');
-  const resizedImageOptions = useContent({
-    source: (customFields.lazyLoad || isAdmin) ? 'resize-image-api-client' : 'resize-image-api',
-    query: { raw_image_url: customFields.imageURL, 'arc-site': arcSite },
-  });
-  const ratios = ratiosFor('SM', customFields.imageRatio);
-
-  const promoContainersStyles = {
-    containerClass: getPromoStyle(imagePosition, 'container'),
-    headlineClass: showImage
-      ? 'col-sm-xl-8'
-      : 'col-sm-xl-12',
-    imageClass: 'col-sm-xl-4',
-  };
+  const { isAdmin } = useFusionContext();
 
-  const renderWithLink = useCallback((element, props, attributes) => (
-    <a
-      href={customFields.linkURL || '#'}
-      className={(props && props.className) || ''}
-      target={customFields.newTab ? '_blank' : '_self'}
-      rel={customFields.newTab ? 'noreferrer noopener' : ''}
-      onClick={!customFields.linkURL ? (evt) => {
-        evt.preventDefault();
-      } : undefined}
-      {...attributes}
-    >
-      {element}
-    </a>
-  ), [customFields.linkURL, customFields.newTab]);
+  const imagePosition = customFields?.imagePosition || 'right';
+  const headlineMarginClass = SmallPromoStyles(imagePosition, 'headlineMargin');
 
-  const headline = customFields.showHeadline && customFields.headline
+  const headlineOutput = customFields.showHeadline && customFields.headline
```

Comment:
small nit: can we make these ternaries with null as the negative case just in case we get some odd boolean coercion https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx